__tests__/server.test.js

```diff
@@ -24,13 +24,112 @@ describe('Person Route', () => {
 
 //REST route tests
 
-//Food route
-describe('REST route tests', () => {
+describe('Food route tests', () => {
```

Comment:
Test the post routes first so that way you can set up a 'mock' database and use it to test your other REST methods for the Food route